usaspending_api/disaster/tests/integration/test_object_class_spending_award.py

```diff
@@ -52,18 +60,22 @@ def test_object_class_counts_awards(client, faba_with_object_class_and_two_award
 
 @pytest.mark.django_db
 def test_object_class_groups_by_object_classes(
-    client, faba_with_two_object_classes_and_two_awards, monkeypatch, helpers
+    client, elasticsearch_account_index, faba_with_two_object_classes_and_two_awards, monkeypatch, helpers
 ):
+    setup_elasticsearch_test(monkeypatch, elasticsearch_account_index)
     helpers.patch_datetime_now(monkeypatch, 2022, 12, 31)
     helpers.reset_dabs_cache()
 
-    resp = helpers.post_for_spending_endpoint(client, url, def_codes=["M"], spending_type="award")
+    resp = helpers.post_for_spending_endpoint(client, url, def_codes=["M", "N"], spending_type="award")
```

Comment:
Concerned about this change being necessary if the `faba_with_two_object_classes_and_two_awards` fixture changes are not obviously tied